src/server/webserver.ts

```diff
@@ -208,7 +208,7 @@ export class WebServer {
             }
           } else {
             // If TLS is going to be used, setup a TLS socket
-            log.info('TLS Enabled!')
+            log.debug('TLS Enabled!')
```

Comment:
seems like this one can remain info -- its only every logged at most once during start up right?

Reply:
yes it looks like it is called in the on connection for the ws